website/docs/r/spring_cloud_service.html.markdown

```diff
@@ -75,6 +77,20 @@ The following arguments are supported:
 
 The `config_server_git_setting` block supports the following:
 
+* `service_runtime_subnet_id` - (Required) Specifies the subnet id to host Azure Spring Cloud Service Runtime.
```

Comment:
what does this mean in practice?

Reply:
"(Required) Specifies the full qualified resource id of an Azure Virtual Network Subnet, which will be used to host the Service Runtime components of the Azure Spring Cloud Service Instance to be created."

Reply:
we avoid the phrase "Resource ID" since it's ambiguous between Azure and Terraform - so perhaps:

> * `service_runtime_subnet_id` - (Required) Specifies the ID of the Subnet where the Service Runtime components of the Spring Cloud Service will exist. Changing this forces a new resource to be created.

Reply:
done